tests/unittests/CoreGraphics.drawing/CGContextDrawing_DrawingOptionsTests.cpp

```diff
@@ -85,8 +85,8 @@ class CGDrawLineDash : public WhiteBackgroundTest<>, public ::testing::WithParam
     CFStringRef CreateOutputFilename() {
         PhaseDashCount dashes = GetParam();
 
-        auto dashString = woc::MakeStrongCF<CFMutableStringRef>(
-            CFStringCreateMutableCopy(NULL, 0, CFStringCreateWithFormat(nullptr, nullptr, CFSTR("%g"), dashes.dashPattern[0])));
+        auto startString = woc::MakeStrongCF<CFStringRef>(CFStringCreateWithFormat(nullptr, nullptr, CFSTR("%g"), dashes.dashPattern[0]));
+        auto dashString = woc::MakeStrongCF<CFMutableStringRef>(CFStringCreateMutableCopy(NULL, 0, startString));
```

Comment:
you could avoid this if you wanted to by creating an empty mutable string and using CFStringAppendFormat like you already are